docs/en/latest/plugins/file-logger.md

```diff
@@ -43,10 +43,10 @@ The `file-logger` Plugin is used to push log streams to a specific location.
 
 ## Attributes
 
-| Name | Type   | Required | Description   |
-| ---- | ------ | -------- | ------------- |
+| Name | Type   | Required | Description                                                                                                                                                                                                                        |
+| ---- | ------ | -------- |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | path | string | True     | Log file path. |
-| include_resp_body      | boolean | False     | When set to `true` includes the response body in the log file.                                                                                                                                                                |
+| include_resp_body      | boolean | False     | When set to `true` includes the response body in the log file. If format is set, you can use the variable $resp_body to get the response body.                                                                                     |
```

Comment:
Would be better to put it in https://github.com/apache/apisix/blob/master/docs/en/latest/apisix-variable.md, instead of modifying each logger's doc.
Note that we need to mean that this variable is only for logger and requires the logger to have a special configuration.

Reply:
+1.